src/scene/light.js

```diff
@@ -393,6 +394,9 @@ class Light {
 
         this._outerConeAngle = value;
         this._outerConeAngleCos = Math.cos(value * Math.PI / 180);
+        if (this._scene?.physicalUnits) {
+            this._updateFinalColor();
+        }
```

Comment:
one possible issue here is if this property is set first, and then the scene is switched to use physical units. Or in general, if scene is switched to use physical units after the lights were already created. But perhaps that's not a common case, as a solution is not straightforward.

Reply:
That's a tricky one to do, the scene is created by the application and not the user. We could do automatic job with some 'version'  property on the light .. which would be compared each frame the the 'version' stored on the scene. If scene.physicalUnits changes, we'd increment the version on the scene. When culling lights during rendering, if the versions don't match, we'd evaluate final color again. But it's not very nice solution.

Reply:
actually. Maybe the light can have internal flag 'usesPhysicalUnits' or something similar. During rendering, inside forwardRenderer.cullLights, when the light is visible, and its usesPhysicalUnits does not match scene.physicalUnits, we'd force evaluate the light's final color.